app/dev_mode/views.py

```diff
@@ -74,7 +75,7 @@ def extract_eq_id_and_form_type(schema_name):
 
 
 def create_payload(user, exp_time, eq_id, period_str, period_id, form_type, collection_exercise_sid, ref_p_start_date,
```

Comment:
Can we change the method parameters to be the form so it doesn't have the be the world record holder for method parameters